### ISSUE TITLE
Add network selector to fixed section of mobile navbar

### ISSUE THREAD

**Issue by dvnahuel** [hemilabs/ui-monorepo#1037]
On mobile views, the menu navbar currently lacks the network selector in the fixed section. To improve accessibility, we need to add the network selector to the fixed section of the mobile navbar.

Designs [here](https://www.figma.com/design/4fVd9wneclsvYDYD95ApZ9/Hemi-Portal?node-id=4263-47090&t=YD2N3BrvXjoNM7wm-1)
Proposed implementation

<img width="371" alt="Image" src="https://github.com/user-attachments/assets/fde11fa2-c6aa-422a-97e6-28d0e315e5c2" />

**Comment by qgatssdev** [comment]
@gndelia Is this something I can fix? Don't have access to the figma
